apps/bbsync/query.py

```diff
@@ -136,11 +136,11 @@ def generate_unconditional(self):
         """
         generate query attributes not requiring conditional processing
         """
-        self._query["cf_release_notes"] = self.flaw.summary
+        self._query["cf_release_notes"] = self.flaw.cve_description
         self._query["severity"] = self.IMPACT_TO_SEVERITY_PRIORITY[self.flaw.impact]
         self._query["priority"] = self.IMPACT_TO_SEVERITY_PRIORITY[self.flaw.impact]
 
-    def generate_summary(self):
+    def generate_cve_description(self):
```

Comment:
I am thinking about whether this function should actually be renamed or not, as basically this serves merely for generating the data for Bugzilla and in Bugzilla, this field will still be named `summary` :thinking: Maybe just updating the doc text of that function and clearly state the `cve_description` <-> `summary` info?